dhcpv6/ztp/parse_remote_id.go

```diff
@@ -0,0 +1,84 @@
+package ztpv6
```

Comment:
package name is different from directory name

Reply:
Do you have a preference on ztpv6 vs ztp for directory/package name? I changed package to ztp in my last update but that may not be the best idea.

Reply:
I would use `ztpv6` to differentiate from the v4 package.

Reply:
I agree that it would be confusing to have the same name for both. At the same time I don't like having "ztp" for v4 and "ztpv6" for v6, because it's inconsistent

Reply:
However I don't see an easy solution without renaming ztp to ztpv6